ReactNativeClient/lib/components/screens/note.js

```diff
@@ -744,7 +744,10 @@ class NoteScreenComponent extends BaseScreenComponent {
 	titleTextInput_contentSizeChange(event) {
 		if (!this.enableMultilineTitle_) return;
 
-		const height = event.nativeEvent.contentSize.height;
+		let height = event.nativeEvent.contentSize.height;
+
+		// this is to limit the size of text input, so that it can't cover full screen
+		height  = height > 100 ? 100 : height;
```

Comment:
Could you show a screenshot of how it looked before and after?

Reply:
Sure sir, before it will take full screen and body of the note was not even visible. and we couldn't even scroll.
![Screenshot_1584983962](https://user-images.githubusercontent.com/27751740/77372421-ae472000-6d8b-11ea-9a64-b6e40ca5ce24.png)